public/js/main-template.js

```diff
@@ -12,93 +12,122 @@ $(document).ready(function () {
     asideBarRight = document.getElementById("AsideBarRight");
     fieldContainer = document.getElementById("field-container");
     installcontainer = document.getElementById("install-container");
+   if (document.getElementById("pannelone")) {

```

Comment:
this way can be more reusable and more redable.
```suggestion
   const pannelOne=document.getElementById("pannelone");
   if (pannelOne) {
   pannelOne.classList.toggle("Active");
   }

```